Tests/CommandsTests/SwiftToolTests.swift

```diff
@@ -237,6 +237,7 @@ extension SwiftTool {
         return try SwiftTool(
             outputStream: outputStream,
             options: options,
+            shouldInstallSignalHandlers: false,
```

Comment:
feels like this belongs into options

Reply:
Moved to `ToolWorkspaceConfiguration` for now per my comment above.